mlx/xunit2rst.mako

```diff
@@ -109,5 +122,10 @@ ${generate_body(failure.get('message'), failure.get('type'))}
 
         % endfor
     %endfor
+% endif
+% if add_links:
+    Link to `log file with details for ${test_name} \
```

Comment:
shouldnt we rather have `:ext_reference:` as a link option?

Reply:
How will that look?  If it can look like "log file with details of this test case", I would be OK.  If it is an expanded URL or "external link".
Does that cause the external link also to be an item?  That sounds too much.

Reply:
Same as we have links to Polarion, where ID in the link is clickable and leads to the Polarion WorkItem. So this would be just another link in the links section and not in description as it is now.

Reply:
What do you want as string representation of this external relationship? What do you want as text string for the hyperlink?

Reply:
`${log_file}` goes to conf.py and `${"s1-" if indexes[0] else ""}s${indexes[0] if indexes[0] else 1}-t${indexes[1]}` is value.

So I assume id would be `:ext_robotframeworklog:` then string representation can be anything user defines? But I agree this brings in configuration of the traceability extension, but then again, we already have dependency on it.

Reply:
Done in d9157de50dc9d296aae4a50be2115855131d674f

Extract from example documentation: 

![image](https://user-images.githubusercontent.com/28319872/150806067-ffe8638f-a11b-4639-96d4-8c6d9ece1f32.png)

Reply:
To me this seems better. And readme nicely describes the process of how you add that external reference to the `conf.py`